model/project.go

```diff
@@ -71,7 +71,8 @@ type Project struct {
 	// The below fields can be set for the ProjectRef struct on the project page, or in the project config yaml.
 	// Values for the below fields set on this struct when TranslateProject is called for the project parser will
 	// take precedence over the project page and will be the configs used for a given project during runtime.
-	DeactivatePrevious bool `yaml:"deactivate_previous" bson:"deactivate_previous,omitempty"`
+	DeactivatePrevious bool               `yaml:"deactivate_previous" bson:"deactivate_previous,omitempty"`
+	WorkstationConfig  *WorkstationConfig `yaml:"workstation_config,omitempty" bson:"workstation_config,omitempty"`
```

Comment:
Similar to the previous PR, I'm not sure why we need this in Project.

Reply:
Ok, since they are not used I've removed properties added to the Project that aren't referenced anywhere.